keycloak/keycloak-admin-ui
# Add UI and partial functionality for creating SAML identity providers

## Motivation
https://issues.redhat.com/browse/APPDUX-1027

## Brief Description
Adds most UI elements for creating/editing a SAML v2.0 identity provider, and preliminary CRUD functionality for a basic SAML provider. Note that not all functionality is working - some fields aren't wired, etc. This is the first of several PRs to come in order avoid one huge PR.

## Verification Steps
Although the intention of this PR is to get most of the UI elements, strings, help, etc into the code, you should be able to successfully create/edit/delete a SAML provider using these steps:
1. Select **Identity Providers** and click the **SAML v2.0** card (if no other identity providers exist), or if other providers exist, click **Add provider** and select **SAML v2.0**. 
2. Set the **Use entity descriptor** to **Off**.
3. Enter a secure URL in the **Single Sign-On service URL**, e.g. "**https://localhost.8180/auth/realms/sso**".
4. Click **Add**. 

Once created, you should be able to view expanded settings, use the jump links, etc. All help strings should be valid, and you should be able to edit some fields and Save (not all are working). You should also be able to Delete.  

## Checklist:
- [x] Cypress tests have been created/updated

## Additional Notes
None
